bundle/pom.xml

```diff
@@ -209,30 +209,41 @@
 
       <!-- Nodejs Maven Plugin to run the templates compilation and tests -->
       <plugin>
-        <groupId>io.wcm.maven.plugins</groupId>
-        <artifactId>nodejs-maven-plugin</artifactId>
+        <groupId>com.github.eirslett</groupId>
+        <artifactId>frontend-maven-plugin</artifactId>
         <executions>
           <execution>
-            <phase>generate-sources</phase>
-            <goals><goal>run</goal></goals>
+            <id>install node and npm</id>
+            <goals>
+              <goal>install-node-and-npm</goal>
+            </goals>
+            <phase>process-resources</phase>
+            <configuration>
+              <nodeVersion>v20.11.1</nodeVersion>
+              <npmVersion>10.2.4</npmVersion>
```

Comment:
no need to explicitly add that, it is derived from the nodeVersion usually!

Reply:
good point - that was not the case in the past (probably long time ago), but it works fine now, thanks